src/features/searchResults/mockData.ts

```diff
@@ -0,0 +1,78 @@
+import { TMemberCard } from 'features/members/MemberCard'
+import { TNFTCard } from '../../common/components/NFTCard'
+import mockAvatar from '../../common/assets/images/avatar2-placeholder.png'
+import image from '../../common/assets/images/nft-card-placeholder.png'
```

Comment:
Can we change the variable name to `nftCard`?

Reply:
We discussed as "T". Need to change again?

Reply:
@ihor-stack I think we just need change the name from `image` to `nftCard` is ok.